### DISCUSSION TITLE
Use ActiveRecord.after_all_transactions_commit

### DISCUSSION THREAD

**Discussion by such** [sidekiq/sidekiq#6418] [Ideas]
Starting from Rails 7.2, it appears it's now possible to replace the use of `after_commit_everywhere` by `ActiveRecord.after_all_transactions_commit` for `transactional_push!`. https://edgeguides.rubyonrails.org/7_2_release_notes.html#per-transaction-commit-and-rollback-callbacks

Has this been considered already?

**Comment by mperham** [comment]
Not yet, because it’s brand new. PRs welcome.
